recipes/hatch-ci/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "hatch-ci" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace("-", "_") }}-{{ version }}.tar.gz
+  sha256: 846efdcc9f2adaa2da9074e1901ce100ec3830143349ba1f5befa911b6a39a3c
+  patches:
+    # this is a one-off patch to allow to build the package (because it is a recursive dependecy)
+    - conda.patch    
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - hatchling
+    - typing-extensions
+    - jinja2
+  run:
+    - python
+    - hatchling
+    - typing-extensions
+    - jinja2
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
```

Comment:
Please remove the comments from the example recipe.

Reply:
done